docs/releases.md

```diff
@@ -14,61 +14,21 @@ We use [Semantic Versioning](https://semver.org/#semantic-versioning-200).
    This will update the `package.json` version, create a commit and add
    an annotated version tag.
 
-## 2. Update the changelog
+3. Push your tag and branch: `git push origin <tag> <branch>`
 
-We use conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
-
-1. Run
-
-   ```shell
-   $  gulp changelog
-   ```
-
-   This outputs some suggested changelog content based on `git` history
-
-2. Edit `CHANGELOG.md` and commit changes
-
-   Add content to `CHANGELOG.md` above the last set of changes. Edit the
-   suggested output (above) to be useful and meaningful.
-
-## 3. Squash, update tag, push
-
-1. Squash into a single commit
-
-   Squash or amend the changelog edits into the same commit as the `package.json`
-   version bump. You should now have a single commit on your branch.
-
-2. Move the version tag
-
-   You've created a new commit with the combined version/changelog changes,
-   and the version tag needs to be updated to point at the hash for this combined commit:
-
-   ```shell
-   $ git tag -af <tag> <commit>
-   ```
-
-3. Push
-
-   Push both the branch and the tag
-
-   ```shell
-   $ git push origin <tag> <branch>
-   ```
-
-   Where `<branch>` can be `HEAD` assuming you have the release branch checked out locally.
-
-## 4. Merge and release
+## 2. Merge and release on GitHub
 
 1. Go through the PR/merge cycle for the version-update branch
 2. Create a [new GitHub release](https://github.com/hypothesis/frontend-shared/releases/new/)
 
    - Title: use the tag name
-   - Message/Description: Paste in changelog changes
+   - Click the `Auto-generate release notes` button to generate release notes.
+     Edit as desired.
```

Comment:
I'd like to adjust this to emphasize here that:

1. Proper release notes are required, and not an optional step.
2. It is important to manually refine the release notes to highlight breaking changes. The format advice from https://keepachangelog.com/en/1.0.0/ is still useful even if we are not keeping the changelog as a file in the source tree